packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -760,6 +760,9 @@ def _check_sync(
                         f"local height == remote == {local_height}; Sync complete..."
                     )
                     self.round_sequence.end_sync()
+                    # we set the block stall deadline here because we pinged the /status endpoint
+                    # and received a response from tm, which means that the communication is fine
+                    self.round_sequence.set_block_stall_deadline()
```

Comment:
This is the fix. The rest is mostly unrelated cleanup, done in 4b3955e632a2584154ff04bcd7f70baf96ecc02a.